packages/@okta/vuepress-theme-prose/global-components/EventTypes.vue

```diff
@@ -39,7 +39,9 @@
       SmartLink: () => import("../components/SmartLink"),
     },
     created() {
-      this.eventTypes = eventTypes.versions[1].eventTypes.filter(eventType => !eventType.beta)
+      this.eventTypes = eventTypes.versions
+        .find(version => version.version == "V2").eventTypes
+        .filter(eventType => !eventType.beta && !eventType.internal)
```

Comment:
We want to include `internal` in the file for other use cases, but not in the UI.